themes/opentermsarchive/layouts/alias.html

```diff
@@ -1,11 +1,32 @@
 <!doctype html>
 <html lang="{{ site.Language.LanguageCode }}">
   <head>
-    <title>{{ .Permalink }}</title>
+    <title>{{ substr .Permalink 0 120 }}</title>
```

Comment:
Add comment to explain why

Reply:
As we saw together, I'm putting a translatable text in its place.